server/container_create_linux.go

```diff
@@ -802,13 +802,17 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 	if containerIDMappings == nil {
 		rootPair = idtools.IDPair{UID: 0, GID: 0}
 	}
+
+	etc := filepath.Join(mountPoint, "/etc")
+	// create the `/etc` folder only when it doesn't exist
+	if _, err := os.Stat(etc); err != nil && os.IsNotExist(err) {
+		if err := idtools.MkdirAllAndChown(filepath.Join(etc, "mtab"), 0o755, rootPair); err != nil {
```

Comment:
wait 
/hold
 this should be `MkdirAllAndChown(etc...`

Reply:
Fixed